fixtures/fail-versions.travis

```diff
@@ -1,3 +1,4 @@
 # FAILURE
-# *ERROR* servant-client-core is missing tested-with annotations for: ghc-7.8.1,ghc-7.8.2,ghc-7.8.3,ghc-7.8.4
-# *ERROR* servant-foreign is missing tested-with annotations for: ghc-7.8.1,ghc-7.8.2,ghc-7.8.3,ghc-7.8.4
+# *ERROR* servant-client-core is missing tested-with annotations for: ghc-7.8.1, ghc-7.8.2, ghc-7.8.3, ghc-7.8.4
+# *ERROR* servant-foreign is missing tested-with annotations for: ghc-7.8.1, ghc-7.8.2, ghc-7.8.3, ghc-7.8.4
+(Consider option --jobs-selection any)
```

Comment:
You break (unspoken) invariant that errors are single lines.

Perfectly, *Consider...* text would be output with `putStrLnInfo`. That would be easier from call site of `checkVersionsInfo` (unfortunately you have to do it in three places)

Reply:
Is it worthwhile to drill up the API for that?
* `checkVersionsUniform` seems to be the easiest place to add the "Consider" info to the error.
* One level up, in returning from `checkVersions`, I cannot attach this info, because the error strings could come from `checkVersionsAny`, in which case I should not attach the info.
* I could follow your suggestion to try to attach the info at all three call sites of `checkVersions`.  But there I would have to query the value of `(cfgTestedWith config)` a second time.  This would water down the abstraction `checkVersions`.
* If it is important to use `putStrLnInfo` instead of sticking the info at the end of the error message, then I'd rather suggest to return a more structured result from `checkVersions`.  Something like
```haskell
data CheckVersionsResult a b
  = CheckVersionsSucceeded 
    { compilerVersions :: S.Set CompilerVersion
    , cvProject :: Project a b Package
    }
  | CheckVersionsFailed
    { cvErrors :: [String]
    , cvInfos :: [String]
    }
```
How would you like that?